### DISCUSSION TITLE
Local admin users

### DISCUSSION THREAD

**Discussion by mareban** [glpi-project/glpi-agent#453] [Ideas]
### Your idea

Is it possible to have an inventory of local admins ? If yes, how to do that and where there are located in the web console please ?

Thanks for your help.

PS : i've seen bug related to local users inventory and try --scan-profiles without success :-(

**Comment by g-bougard** [comment]
Hi @mareban 

no "administrator" attribute is inventoried for users.

`--scan-profile` has nothing to do regarding users inventory. It only permits to scan users profiles for dedicated software installation or dedicated virtualmachines. If you identified a bug, please, just open an issue explaning what is the issue in details (glpi-agent version, OS, etc...) so we can evaluate if it's really a bug.
